app/models/intake.rb

```diff
@@ -319,6 +319,7 @@ def self.searchable_fields
     }
   }
 
+  # Memoize response into instance variable.
```

Comment:
It's not memoized because there's no `@` here. Is that intentional?

(Not a blocker, but it is confusing.)

Reply:
oh, i need to remove this note. I had tried doing something and then it caused issues so i removed it.